packages/daemon/src/app/controller/base.ts

```diff
@@ -2,7 +2,10 @@ import { inject, Context } from 'midway';
 import { ObjectSchema } from 'joi';
 import * as HttpStatus from 'http-status';
 import { ServerSentEmitter } from '../../utils';
-import { LogManager } from '../../service/log-manager';
+import { TraceManager } from '../../service/trace-manager';
+import Debug from 'debug';
+
+const debug = Debug('base.controller');
```

Comment:
The name is too general, it's recommended to use the package name as a prefix.

Reply:
How about `daemon.base-controller`?